keyboards/keebio/rorschach/keymaps/tuesdayjohn/rules.mk

```diff
@@ -0,0 +1,19 @@
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in 
+#   the appropriate keymap folder that will get included automatically
+#
+
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration(+1000)
```

Comment:
You can remove all of these that haven't changed from the keyboard-level `rules.mk`.